map/map_tests/bookmarks_test.cpp

```diff
@@ -1406,7 +1405,8 @@ UNIT_CLASS_TEST(Runner, ExportAll)
   TEST_EQUAL(bmManager.GetBmGroupsIdList().size(), 4, ());
 
   auto categories = bmManager.GetBmGroupsIdList();
-  auto checker = [](BookmarkManager::SharingResult const & result)
+  bmManager.ResetTestLatch();
```

Comment:
Are `ResetTestLatch` calls really necessary?

Reply:
yes, for single usage of latch it is not required